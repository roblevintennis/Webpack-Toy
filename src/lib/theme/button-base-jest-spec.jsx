import React from 'react';
import renderer from 'react-test-renderer';
import Button from './button-base.jsx';

describe('#Button renders', () => {
  it('correctly', () => {
    const tree = renderer
      .create(<Button>Yo Gabba Gabba</Button>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
