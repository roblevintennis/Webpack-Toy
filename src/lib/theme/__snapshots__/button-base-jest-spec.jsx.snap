// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Button renders correctly 1`] = `
<button
  className="base"
>
  Yo Gabba Gabba
</button>
`;
